aiohttp_session/redis_storage.py

```diff
@@ -43,9 +43,13 @@ def save_session(self, request, response, session):
             self.save_cookie(response, key,
                              max_age=session.max_age)
         else:
-            key = str(key)
-            self.save_cookie(response, key,
-                             max_age=session.max_age)
+            if session.empty:
+                self.save_cookie(response, session._mapping,
```

Comment:
Yes. You are correct (about both). Basically anything passed in `cookie_data` field that evaluates to false will result in a deletion of the cookie so passing `''` instead of `session._mapping` would have the same result and would also be consistent usage of the function but I didn't like it (from an appearance perspective), but if you prefer passing an empty string there I have no practical objection to it. Just drop a comment here and I'll push another commit to change it :) Also thank you for looking into it :)